openshift/ubuntu-base/openshift/build.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: ubuntu-base
+labels:
+  app: ${NAME}${SUFFIX}
+  phase: build
+  app.kubernetes.io/name: uvicorn-gunicorn-fastapi
+  app.kubernetes.io/managed-by: template
+  app.kubernetes.io/version: "20.04"
+parameters:
+  - name: NAME
+    value: ubuntu-base
+  - name: SUFFIX
+  - name: VERSION
+    description: Output version
+    value: "20.04"
+  - name: GIT_URL
+    value: https://github.com/bcgov/wps.git
+  - name: GIT_BRANCH
+    value: feature/c-haines-re-factor
```

Comment:
what happens when this branch gets deleted post-merge?

Reply:
Nothing - this build config is run manually to get the ubuntu base image in place. But I have updated it to remove confusion.